packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -110,11 +115,13 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
 
         if (!request.containsKey("modelSchemas") || !request.containsKey(
                         "modelProviderVersion") || request["modelSchemas"] !is List<*>) {
-            postFlutterError(
-                    flutterResult,
-                    FlutterDataStoreFailureMessage.ERROR_CASTING_INPUT_IN_PLATFORM_CODE.toString(),
-                    createErrorMap(Exception(
-                            FlutterDataStoreFailureMessage.AMPLIFY_REQUEST_MALFORMED.toString())))
+            handler.post {
+                postExceptionToFlutterChannel(flutterResult, "DataStoreException",
+                        createSerializedError(ExceptionMessages.missingExceptionMessage,
+                                ExceptionMessages.missingRecoverySuggestion,
+                                "Received invalid request from Dart, modelSchemas or modelProviderVersion" +
+                                        " was not available. Request: " + request.toString()))
```

Comment:
I'd suggest the following (unless I'm misunderstanding the error condition):
`"Received invalid request from Dart; modelSchemas and/or modelProviderVersion are not available.  Request: "`

Reply:
Updated.